cfme/fixtures/parallelizer/__init__.py

```diff
@@ -165,24 +169,19 @@ def __init__(self, config, appliances):
         self.sock.bind(zmq_endpoint)
 
         # clean out old slave config if it exists
-        slave_config = conf_path.join('slave_config.yaml')
-        slave_config.check() and slave_config.remove()
 
-        # write out the slave config
-        conf.runtime['slave_config'] = {
+        self.worker_config = {
             'args': self.config.args,
             'options': dict(  # copy to avoid aliasing
-                self.config.option.__dict__,
+                vars(self.config.option),
                 use_sprout=False,   # Slaves don't use sprout
             ),
             'zmq_endpoint': zmq_endpoint,
+            'appliance_data': getattr(self, "slave_appliances_data", {})
         }
-        if hasattr(self, "slave_appliances_data"):
-            conf.runtime['slave_config']["appliance_data"] = self.slave_appliances_data
-        conf.save('slave_config')
```

Comment:
Q: whether you don't need to update that file now ?

Reply:
the need for the file got completely obliterated by passing the data via json on the cli